plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/local/LocalFileSystemExchangeStorage.java

```diff
@@ -202,6 +206,12 @@ public synchronized void close()
                 sliceInput.close();
                 sliceInput = null;
```

Comment:
I might be missing something obvious. `sliceInput` is closed here, right?

Reply:
Every time `getSliceInput` is called, a new InputStreamSliceInput is created.. so we are not closing every input stream.

Reply:
Oh, then should delete the code of:
```
if (sliceInput != null) {
                sliceInput.close();
                sliceInput = null;
}
```
? It becomes redundant now, right?

Reply:
Done